benchmarks/encoder/config.yaml

```diff
@@ -21,3 +22,6 @@ one-metric:
 many-metrics:
   <<: *base_variant
   nmetrics: 48
+many-traces-v05:
+  <<: *many_traces
+  version: "v0.5"
```

Comment:
Should we have some sort of matrix+templating in place so that we don't have to copy-paste all the configurations just to add a `-v05` suffix and `version: "v0.5"`?

Reply:
That is certainly an option we should consider!